1_spatial.yml

```diff
@@ -58,4 +61,23 @@ targets:
     
   lordville_sites:
     command: get_sites(network_lordville)
-  
\ No newline at end of file
+
+  # Define scope of reservoir modeling data in this repo
+  reservoir_modeling_site_ids:
+    command: c(I(c(
+      Pepacton = 'nhdhr_151957878',
+      Cannonsville = 'nhdhr_120022743')))
+      
+  reservoir_polygons:
+    command: fetch_filter_res_polygons(
+      out_rds = target_name,
+      in_ind = "../lake-temperature-model-prep/1_crosswalk_fetch/out/canonical_lakes_sf.rds.ind",
+      in_repo = I('../lake-temperature-model-prep/'),
+      site_ids = reservoir_modeling_site_ids)
+      
+  reservoir_metadata:
+    command: extract_feature(reservoir_polygons)
+    
+  out_data/01_reservoir_polygons.zip:
+    command: sf_to_zip(zip_filename = target_name, 
+      sf_object = reservoir_polygons, layer_name = I('study_reservoirs'))
```

Comment:
Currently there are no data on the [spatial page](https://www.sciencebase.gov/catalog/item/5f6a285d82ce38aaa244912e). Is that expected?

Reply:
Ugh, no. Science Base was being buggy last night - I'll push those again now

Reply:
Ok the 2 zip files should be there now